aroma/features.py

```diff
@@ -123,10 +129,11 @@ def feature_frequency(melFTmix, TR):
 
     # Load melodic_FTmix file
     FT = np.loadtxt(melFTmix)
+    n_frequencies, _ = FT.shape
```

Comment:
Why not `FT.shape[0]`?

Reply:
I think it's a holdover from when I thought the number of components might come into play in this function. I'll change it.